bemlint/lib/rules/block.js

```diff
@@ -0,0 +1,30 @@
+'use strict';
+
+const {
+  createError,
+} = require('../helpers');
+const { block: massages } = require('./errorMassage.json');
+
+module.exports = {
+  underscore(
+    node,
+    wrapperBlocks,
+    blocks,
+    elements,
+    modifiers,
+  ) {
+    return blocks.map(block => {
+      if (block.includes('_')) {
```

Comment:
```suggestion
      if (block.includes('_')) {
```
Why do you disallow to use `_` in the block name?